provider/ansible.py

```diff
@@ -132,17 +132,17 @@ def python_install():
             LOG_JOB.error("Failed to get available pip binary")
             return False
         install_cmd = '%s install ansible' % pip_bin
+        # Install ansible depended packages that can't be installed
+        # automatically by pip when installing ansible
+        package_list = params.objects("package_list")
```

Comment:
In the way, `sshpass` be missed in upstream by default.
suggest:
```
package_list = params.get("package_list", 'sshpass')
package_list = params.objects("package_list")
```

Reply:
Correct the second line, we can not use `params.objects`, let's split the `package_list`. thanks!

Reply:
@yanan-fu, why don't we set `sshpass` as the default value in cfg, and then we can use `params.objects` to generate a list.

BTW, `params.get_list` can generate a list and also set a default value.

Reply:
Same result, i am ok with any resolution.  As long as there is default value, it is ok for me.
Thanks!